praw/objects.py

```diff
@@ -1301,7 +1301,7 @@ def short_link(self):
         return urljoin(self.reddit_session.config.short_domain, self.id)
 
     @restrict_access(scope='modposts')
-    def sticky(self, num=2):
+    def sticky(self, bottom=True):
```

Comment:
The parameter name should be updated. Also we should probably explain what actually happens, e.g., setting bottom when there is no top will make it the top.